articles/how-to-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/index.md

```diff
@@ -0,0 +1,131 @@
+these data for various projects.
+
+### Prerequisites
+- [Matlab](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [matlab](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+- Mobile matlab installed on your phone.
+
+To install Matlab mobile app on your phone, go to the play store or AppStore and search `Matlab`. Once you locate this, click on the install, and you
+shall have installed Matlab into your phone.
+
+### How to use mobile phone's Matlab to take photos of images
+You can access the camera to take an image or video manually. This image or video you can later use in your program for analysis. To access the camera, 
+- Open the Matlab on your mobile phone.
+> Note that mobile Matlab works online, and you must have an internet connection to use it.
+- Click on the three dashes on the top-left of the Matlab window.
+![location of the dashes](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_one.jpeg)
+- Click on the file section.
+![location of the files](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_two.jpeg)
+- Click on the `+` sign at the top-right corner.
+- Click on the take a photo to take a photo of your image.
+![tab for taking the photo](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_three.jpeg)
+
+Using the camera on the Matlab mobile app is easy and intuitive, just like you would on most applications. After taking an image, it is uploaded to a 
+Matlab drive to sync all your files and data. The image is uploaded in the `.jpeg` format with an autogenerated name. Since you are going to use this image,
+you can rename it to be something meaningful. To rename the image, we long-press it, click on the `rename` tab, and give it a name you wish. After giving it 
+a new name, you click `ok`. 
+Let us now go back to the command window and begin writing some codes. We will view the image using the `imread` function and then use the `imshow` function 
+to view it. 
+```Matlab
+I = imread('filename.jpg');
+imshow(I)
+```
+![showing our captured image](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_four.jpeg)
+After accessing your image, you can do anything you wish with the image. For example, you can change the image to grayscale and many more options. The code is;
+```Matlab
+B = rgb2gray(I);
+imshow(B)
+```
+![the grayscale image](/howto-collect-sensor-data-on-your-iphone-or-android-device-using-matlab/engineering-education/sensor_five.jpeg)
+As you can see, it was easy to use our mobile Matlab to acquire the image and perform some quick image processing. You can write all your commands in a 
```

Comment:
```suggestion
As you can see, it is easy to use your mobile Matlab to acquire the image and perform some quick image processing. You can write all your commands in a script so that you could use them anytime on your mobile Matlab or PC.

To do this, first, click on the `+` sign on the file section then click on `script.`
```